src/storage-client/src/types/sources.proto

```diff
@@ -222,9 +222,7 @@ message ProtoTestScriptSourceConnection {
 }
 
 message ProtoCounterLoadGenerator {
-    // Must be non-negative,
-    // but kept as int64 to make downstream logic simpler
-    optional int64 max_cardinality = 1;
+    optional uint64 max_cardinality = 1;
```

Comment:
Should this get a new number because the type has changed?

Reply:
These proto files are not stored anywhere so there is no need to clutter them (yet) with new numbers and reserved fields